conf/messages

```diff
@@ -280,8 +280,10 @@ nisp.nirecord.gap.selfemployed.singular = Self-employment: <strong>{0} week</str
 nisp.nirecord.gap.selfemployed.plural = Self-employment: <strong>{0} weeks</strong>
 nisp.nirecord.gap.voluntary.singular = Voluntary: <strong>{0} week</strong>
 nisp.nirecord.gap.voluntary.plural = Voluntary: <strong>{0} weeks</strong>
-nisp.nirecord.gap.whenyouareclaiming.singular = National Insurance credits: <strong>{0} week</strong> (this week may have been added to your record if you were ill/disabled, unemployed, caring for someone full-time or on jury service) 
-nisp.nirecord.gap.whenyouareclaiming.plural= National Insurance credits: <strong>{0} weeks</strong> (these may have been added to your record if you were ill/disabled, unemployed, caring for someone full-time or on jury service) 
+nisp.nirecord.gap.whenyouareclaiming.singular = National Insurance credits: <strong>{0} week</strong>
```

Comment:
Should we remove the {0} and replace it with "1" (for all singular messages)? 

It is only for a single year, so we don't need to pass a value.